stdlib/Dates/docs/src/index.md

```diff
@@ -400,29 +400,29 @@ As a bonus, all period arithmetic objects work directly with ranges:
 
 ```jldoctest
 julia> dr = Date(2014,1,29):Day(1):Date(2014,2,3)
-2014-01-29:1 day:2014-02-03
+Date(2014, 1, 29):1 day:Date(2014, 2, 3)
```

Comment:
These changes seems like an unintended consequence? At least to me it is a step back in readability.

Reply:
No, the idea is to _show_ dates and times in a format that is valid input syntax. The fact that the step doesn't quite use valid input syntax is the remaining issue here—if that was shown as `Day(1)` instead then this whole output would be valid input syntax. Printing date ranges could use a more human-readable format instead of a valid input syntax.

Reply:
Ok, placed the comment on the wrong line; do we want the new printing from the `collect` on the next lines?

Reply:
Not sure, I guess we can use either format there. This seems fine to me though.